### ISSUE TITLE
kebab-case for the CSS property names not working

### ISSUE THREAD

**Issue by wsw70** [vuejs/vuejs.org#576]
The [documentation](https://vuejs.org/v2/guide/class-and-style.html#Binding-Inline-Styles) claims that

> You can use either camelCase or kebab-case for the CSS property names

This is not the case. As of VueJS 2.0.5 only camelCase is properly handled, kebab-case silently fails (and nothing is rendered).

**Comment by LinusBorg** [comment]
Since I assume you don'T want to change the documentation, but have Vue fixed, please post your bug report in the right repository (github.com/vuejs/vue) - with a reproduction.

**Comment by wsw70** [comment]
@LinusBorg I actually forked the documentation to suggest a chnage but then though that it might be better to have this raised as a bug since the handling of both cases was, I believe, intended intended (as opposed to a documentation bug). I will open the bug where you suggested to.

**Comment by LinusBorg** [comment]
@wsw70 have you seen that it works in my demo? Sure this is a bug?

**Comment by wsw70** [comment]
@LinusBorg : yes, you are right - I realized that yesterday night when doing a JSFifddle to report the bug. I did not know about the hyphens (I come from the python world where the hyphen is compulsory in any case). I need to learn more about this (it is not the first case where I fall for that trap, also with the dot notation). Thank you.
I was about to suggest that it could be worthwhile to mention this in the docs but OTOH it looks like this is  is something obvious in JS. I found in the meantime a [SO comment to an answer](http://stackoverflow.com/a/35244182/903011) with the same problem and a comment mentions the "bug" I thought I found too.
